### PR TITLE
Release notes for 24.12b1

### PR DESCRIPTION
**Pull request by neilweinstock**
This was supposed to be a draft PR but I fumbled.  Don't merge yet until there's been time for review.

### CODE REVIEW

File: ReleaseNotes.md [hunk 1]
```diff
@@ -13,6 +13,104 @@ Release Notes
 
 </div>
 
+<div id="24.12">
```
Comment (SiboVG) [inline ReleaseNotes.md:16]: change to 24.12.beta.01

File: ReleaseNotes.md [hunk 2]
```diff
@@ -13,6 +13,104 @@ Release Notes
...
+ 
+## OpenRocket 24.12 beta 1
```
Comment (SiboVG) [inline ReleaseNotes.md:18]: Use the same formatting as the other releases:
```
OpenRocket 24.12.beta.01 (2024-12-XX)
-----------------------------------------
```

**Review by SiboVG — CHANGES_REQUESTED** [review]

File: ReleaseNotes.md [hunk 3]
```diff
@@ -13,6 +13,104 @@ Release Notes
...
+* Fix Mass Object radial rotation in 3D view (fixes 2550)
+* Improve texture select combobox operation
+
+### Command Line Options
+* Choose user-defined component preset locations (fixes 1081)
+* Suppress preset and motor loading at startup (fixes 1579)
+
+### Project
+* **Move project documentation to Sphinx**
+* **Switched build system from And to Gradle**
```
Comment (SiboVG) [inline ReleaseNotes.md:102]: Ant*

File: ReleaseNotes.md [hunk 4]
```diff
@@ -13,6 +13,104 @@ Release Notes
...
+* **Multi-level wind input** (fixes 922, 2060, 2558)
+* Sync wind speed, deviation, and turbulence widgets together in sim settings (fixes 2388)
+* Add South/West units for latitude and longitude of launch site (fixes 2178)
+* Improved accuracy of thrust calculation
+* Abort sim when recovery deployment occurs under thrust
+* Allow sustainer to tumble before apogee; if under thrust, abort sim
+* Fire outdated sim on stage rename (fixes 2532)
+* Added "aborted" status mark to simulations
+* Don't show vertical acceleration as negative before liftoff
+
```
Comment (SiboVG) [inline ReleaseNotes.md:43]: Also add "Clamp compressibility correction factor to avoid singularity (#2570)"?

**Comment by SiboVG** [comment]
Thanks @neilweinstock! Always fun to see an overview of all the changes :)

**Comment by neilweinstock** [comment]
All changes made, thanks for the quick review!

**Comment by neilweinstock** [comment]
> Thanks @neilweinstock! Always fun to see an overview of all the changes :)

It's quite a lot!

File: ReleaseNotes.md [hunk 5]
```diff
@@ -13,6 +13,106 @@ Release Notes
...
+* **Multi-level wind input** (fixes 922, 2060, 2558)
+* Sync wind speed, deviation, and turbulence widgets together in sim settings (fixes 2388)
+* Add South/West units for latitude and longitude of launch site (fixes 2178)
+* Improved accuracy of thrust calculation
+* Abort sim when recovery deployment occurs under thrust
+* Allow sustainer to tumble before apogee; if under thrust, abort sim
+* Fire outdated sim on stage rename (fixes 2532)
+* Added "aborted" status mark to simulations
+* Don't show vertical acceleration as negative before liftoff
+* Clamp compressibility correction factor to avoid singularity (fixes 2570)
```
Comment (SiboVG) [inline ReleaseNotes.md:44]: 2570 is the ID of the PR, there is no issue related to it.

**Review by SiboVG — APPROVED** [review]

**Comment by neilweinstock** [comment]
We have to wait to fill in the release date, right?  I can't remember if we put dates on the beta releases.

File: ReleaseNotes.md [hunk 6]
```diff
@@ -13,6 +13,106 @@ Release Notes
...
+* Improved accuracy of thrust calculation
+* Abort sim when recovery deployment occurs under thrust
+* Allow sustainer to tumble before apogee; if under thrust, abort sim
+* Fire outdated sim on stage rename (fixes 2532)
+* Added "aborted" status mark to simulations
+* Don't show vertical acceleration as negative before liftoff
+* Clamp compressibility correction factor to avoid singularity
+
+#### Warnings
+* **Add SIM_ABORT flight even type, instead of throwing exceptions**
```
Comment (JoePfeiffer) [inline ReleaseNotes.md:47]: even -> event

File: ReleaseNotes.md [hunk 7]
```diff
@@ -13,6 +13,106 @@ Release Notes
...
+* Don't warn about large angle of attack when we start to tumble
+* Don't set open airframe warning if rocket is unstable (and improve wording)
```
Comment (JoePfeiffer) [inline ReleaseNotes.md:49]: Don't set open airframe warning on booster stages if either they're about to deploy a recovery device or they're unstable.

**Review by JoePfeiffer — APPROVED** [review]

**Pull request merged**